### ISSUE TITLE
ImageDatagen for autoencoders

### ISSUE THREAD

**Issue by Al-yhuwert** [msminhas93/KerasImageDatagenTutorial#2]
Hi,
Thank you for this tutorial, it was very helpful! 

I'm using the ImageDatagenerator for an autoencoder, when using the `class_mode='input'` the input image is also the label.

The thing here is that I want a different image to be the label, how can I do that??
If I do something like this:

` autoencoder.fit(train_it, train_it_2, steps_per_epoch= math.ceil(train_it.samples//batch_size),
                 epochs=number_epochs, validation_data=(val_it, val_it_2), validation_steps=math.ceil(val_it.samples//batch_size))`

I get an error that says that `y` argument is not supported when using `keras.utils.Sequence` as input.

Could you help me?
